service/resource/configmap/resource_test.go

```diff
@@ -697,93 +695,12 @@ func Test_Resource_CloudConfig_GetUpdateState(t *testing.T) {
 			},
 			CurrentState:                         []*apiv1.ConfigMap{},
 			DesiredState:                         []*apiv1.ConfigMap{},
-			ExpectedConfigMapsToCreate:           nil,
-			ExpectedConfigMapsToDelete:           nil,
-			ExpectedConfigMapsToUpdate:           nil,
-			ExpectedMessageContextConfigMapNames: nil,
-		},
-
-		// Test 2, in case current state and desired state are equal the create,
-		// delete and update state should be empty.
```

Comment:
Some cases for the configmap and deployment tests are removed because we removed the support of resource deletion during updates.